packages/fondue/src/components/Modal/Modal.tsx

```diff
@@ -74,7 +72,7 @@ const ModalComponent = memo((props: ModalProps): ReactElement => {
             exit="exit"
             style={{
                 background: 'rgba(0, 0, 0, .5)',
-                zIndex,
+                ...(zIndex ? { zIndex } : {}),
```

Comment:
Is there a reason why we change that line?
if not set, it will be `undefined` and then it won't be added

Reply:
I usually don't trust in magic. But sure. 👍